Editor/meshoptimizer/clusterizer.cpp

```diff
@@ -1,10 +1,10 @@
 // This file is part of meshoptimizer library; see meshoptimizer.h for version/license details
 #include "meshoptimizer.h"
 
-#include <assert.h>
-#include <float.h>
-#include <math.h>
-#include <string.h>
+#include <cassert>
```

Comment:
This is meshoptimizer library, no use to change these because changes will be lost when library is updated. Why is this a warning anyway?

Reply:
This was more the IDE (clion) suggestion, saying that using the c headers directly in C++ is deprecated we should use <clibrary> instead. Probably doesn't make a difference at all, since all headers are most likely guarded for c++ inclusion. Didn't know this was a library, I'll remove header changes from all libraries.